src/sentry/models/groupregression.py

```diff
@@ -0,0 +1,23 @@
+from django.db import models
+from django.utils import timezone
+
+from sentry.db.models import FlexibleForeignKey, Model, sane_repr
+
+
+class GroupRegression(Model):
+    """
+    Describes when a group was marked as a regression.
+    """
+
+    __include_in_export__ = False
+
+    group = FlexibleForeignKey("sentry.Group", unique=True)
```

Comment:
Do we definitely want this to be unique? Wasn't sure if we wanted this as a history of all regressions for the group. Is there a design spec for this project anywhere?

Reply:
Yeah, I think this shouldn't be unique since a group can regress multiple times..

Slack'd a notion doc over